README.md

```diff
@@ -588,6 +602,45 @@ helmut.log
 Junit XML file
 ```
 
+# run-ucc-modinput-tests 
+
+**Description**
+
+- This stage does the setup for executing Modinput tests using [ucc modinput tests framework](https://github.com/splunk/addonfactory-ucc-test) and reports the results
+- It is possible to parallelize Modinput tests execution by using pytest markers. 
+  To do so, one must specify `ucc-modinput-marker` parameter in buid-test-release.yml as in [example](https://github.com/splunk/splunk-add-on-for-google-cloud-platform/blob/34abcf2780d8f223f292c9c2fcc5835b71a8de99/.github/workflows/build-test-release.yml#L34).
+  Markers must be created prior and each test case must be marked (check the following references: [ref1](https://github.com/splunk/splunk-add-on-for-google-cloud-platform/blob/34abcf2780d8f223f292c9c2fcc5835b71a8de99/tests/ucc_modinput_functional/markers.py), 
+[ref2](https://github.com/splunk/splunk-add-on-for-google-cloud-platform/blob/34abcf2780d8f223f292c9c2fcc5835b71a8de99/tests/ucc_modinput_functional/test_google_cloud_rh_settings.py#L19))
+
+**Action used:** 
+- No action
+
+**Pass/fail behaviour**
+
+- The stage is expected to fail only if there are any Modular input test failures defined under tests/ucc_modular_functional
```

Comment:
Do we keep this tests in `tests/ucc_modular_functional` directory?

Reply:
yes.

Reply:
There are some TAs, which has both the legacy and new tests present and to differentiate it, they are put under different directories.

Reply:
It should be` tests/ucc_modinput_functional`. Updated!